galaxy/api/v2/tests/test_collection_view.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from galaxy.main import models
+
+
+class CollectionViewTests(TestCase):
```

Comment:
You can extend `rest_framework.test.APITestCase` class that is inherited from `django.test.TestCase` and includes APIClient instance during setup.